src/components/Create/RulesForm.tsx

```diff
@@ -89,7 +94,7 @@ export default function RulesForm({
             <div>
               <p>{s.description}</p>
               <p style={{ fontSize: '0.7rem', color: colors.text.tertiary }}>
-                Contract address: {s.address}
+                <Trans>Contract address</Trans>: {s.address}
```

Comment:
**issue**: the "syntax" of `key: value` in some languages is different. Here's a decent explanation as to why: https://softwareengineering.stackexchange.com/questions/234546/colons-in-internationalized-ui

This can thus become:

```suggestion
                <Trans>Contract address: {s.address}</Trans>
```

Reply:
> **issue**: the "syntax" of `key: value` in some languages is different. Here's a decent explanation as to why: https://softwareengineering.stackexchange.com/questions/234546/colons-in-internationalized-ui
> 
> This can thus become:

Nice pickup, thanks!